bin/ci-reindex-rds

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -e
+. bin/ci-process-bootstrap
+
+. bin/ci-start-processes
+
+./bin/console rds:reindex
+
+echo "Waiting for empty queues"
```

Comment:
redundant: `./bin/ci-wait-steady-state` does an `echo -n "Waiting for queues to empty..."` immediately.